android/app/src/main/res/values/dimens.xml

```diff
@@ -44,7 +44,7 @@
     <dimen name="heading_text_size">20sp</dimen>
     <dimen name="text_size_expanded_title">22sp</dimen>
     <dimen name="text_size_extra_large">18sp</dimen>
-    <dimen name="text_size_large">16sp</dimen>
+    <dimen name="text_size_large">20sp</dimen>
```

Comment:
Why is large 20sp and extra large 18 sp?

Reply:
reverting back to 16sp